forward_command_controller/src/forward_command_controller.cpp

```diff
@@ -130,17 +103,17 @@ controller_interface::return_type ForwardCommandController::update()
     return controller_interface::return_type::SUCCESS;
   }
 
-  const auto joint_num = (*joint_commands)->data.size();
-  if (joint_num != joint_cmd_handles_.size()) {
+  if ((*joint_commands)->data.size() != command_interfaces_.size()) {
     RCLCPP_ERROR_STREAM_THROTTLE(
-      rclcpp::get_logger(
-        logger_name_),
-      *lifecycle_node_->get_clock(), 1000, "command size does not match number of joints");
+      get_lifecycle_node()->get_logger(),
+      *lifecycle_node_->get_clock(), 1000,
+      "command size (" + std::to_string((*joint_commands)->data.size()) + ") does not match \
+      number of interfaces (" + std::to_string(command_interfaces_.size()) + ")");
```

Comment:
Since the *STREAM* logging macros are being used here, you can skip the string conversion and just stream the log message. Something like this:
```
     "command size (" << (*joint_commands)->data.size() << ") does not match " <<
     "number of interfaces (" << command_interfaces_.size() << ")");
```